stage1/usr_from_src/libnss.mk

```diff
@@ -0,0 +1,18 @@
+$(call setup-stamp-file,UFSN_STAMP)
+LIBNSS_FILES_LINK := libnss_files.so.2
```

Comment:
These variables should have the `UFSN_` prefix, so they will be undefined outside this file.